### ISSUE TITLE
Can't use subcommands (ex: `wallet`) as permission when pushing action with cleos

### ISSUE THREAD

**Issue by DenisCarriere** [EOSIO/eos#8947]
Issue using account names that are the same as commands or subcommands (ex: `-p wallet`) as permission when pushing action with `cleos`

Seems like there's a few command & subcommands which can't be used as account permission.

- `-p wallet`
- `-p create`
- `-p convert`
- `-p set`
- `-p get`
- `-p transfer`
- `-p net`
- `-p sign`
- `-p multisig`
- `-p wrap`
- `-p system`

```bash
$ cleos push action hello hi '["foobar"]' -p wallet
```

**Error message**

```bash
ERROR: RequiredError: Subcommand required
Interact with local wallet
Usage: cleos wallet SUBCOMMAND

Subcommands:
  create                      Create a new wallet locally
  open                        Open an existing wallet
  lock                        Lock wallet
  lock_all                    Lock all unlocked wallets
  unlock                      Unlock wallet
  import                      Import private key into wallet
  remove_key                  Remove key from wallet
  create_key                  Create private key within wallet
  list                        List opened wallets, * = unlocked
  keys                        List of public keys from all unlocked wallets.
  private_keys                List of private keys from an unlocked wallet in wif or PVT_R1 format.
  stop                        Stop keosd.
```

**Comment by revl** [comment]
The fix is in `develop`. It may or may not land in 2.0.

**Comment by revl** [comment]
It was merged into 2.0 as well.

**Comment by DenisCarriere** [comment]
Great thanks 🙏
